test/groovy/tests/TestGenerateAndExecutePipeline.groovy

```diff
@@ -181,6 +192,29 @@ class TestGenerateAndExecutePipeline extends PipelineStepSpecification {
         1 * mockLeaveLock()
     }
 
+    @Unroll
+    def '_isReleaseOrPr returns #isReleaseOrPr when isRelease returns #isReleaseValue and _isPr returns #isPr'() {
+
+        given:
+        def mockIsRelease = createIsReleaseMock()
+        def mockIsPr      = createIsPrMock()
+        when:
+        def actual = script._isReleaseOrPr()
+
+        then:
+        1 * mockIsRelease() >> isReleaseValue
+        1 * mockIsPr() >> isPr
+        then:
+        actual == isReleaseOrPr
+
+        where:
+        isReleaseValue | isPr  | isReleaseOrPr
+        false          | false | false
+        true           | false | true
+        false          | true  | true
+        true           | true  | true
+    }
```

Comment:
Here we have 4 scenarios, it is posible to name each scenario with a description?
example:
scenario                                                 | isReleaseValue | isPr  | isReleaseOrpr 
"This scenario is for happy path....."   | .....

and add #scenario.
Thanks.

Reply:
I believe this comment should be attached to a conversation started earlier [here](https://github.com/gitmlynch/pr_exercise/pull/1#discussion_r298229220)